newsletter.md

```diff
@@ -0,0 +1,14 @@
+
+During the past few months, the team has been working hard to make new features available in the OpenPaaS MVP release.
```

Comment:
I'm not sure to understand if this sentence:
1- is talking about the global work achieve in James during several month/years in order to use it in OpenPaas 
or 
2- is talking about the work done by the James team since the October Newsletter (the last NL talking about James' work)

In the 1st case i would enrich the paragraph with the related features.
In the 2nd case i would say "during the past few weeks" instead of "during the past few months".

thanks in advance for your feedback ;)

Reply:
I guess it is the second case.

Reply:
OK, thank you @chibenwa